File: C/Lv.0/20.c
//삼각형의 완성조건(1)
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// sides_len은 배열 sides의 길이입니다.
int solution(int sides[], size_t sides_len) {
    int answer = 0;
    int max = 0;
    int sum = 0;
    
    for(int i = 0; i < sides_len; i++)
        if (sides[i] > sides[max])
            max = i;
    
    for(int i = 0; i < sides_len; i++)
        if (i != max)
            sum += sides[i];
    
    if(sides[max] < sum)
        answer = 1;
    else
        answer = 2;
    
    return answer;
}

File: Python/Lv.2/20.py
//연속 부분 수열 합의 개수
def solution(elements):
    answer = 0
    
    tmp = set();
    le = len(elements);
    elements *= 2;
    
    for i in range(le):
        for j in range(le):
            tmp.add(sum(elements[j:j + i + 1]));
        
    answer = len(tmp);
    
    return answer